src/app/core/services/api/v2/cup/cup-rating.service.ts

```diff
@@ -37,4 +38,22 @@ export class CupRatingService {
       });
       return this.httpClient.get<PaginatedResponse<CupRatingGroup>>(`${this.cupRatingUrl}-groups`, { params });
    }
+
+   getCupRatingPositionInGroups(search: CupRatingPositionSearch): Observable<CupRatingGroup> {
+      const params: HttpParams = new HttpParams({
+         fromObject: {
+            competition_id: search.competitionId.toString(),
+            position: search.position.toString()
+         }
+      });
+      return this.httpClient.get<CupRatingGroup>(`${this.cupRatingUrl}/rating-position-in-groups`, { params });
+   }
+
+   getCupGroupFurtheranceByPosition(search: CupRatingPositionSearch): Observable<PaginatedResponse<number>> {
+      const params: HttpParams = new HttpParams()
+         .set('competition_id', search.competitionId.toString())
+         .set('position', search.position.toString());
+
+      return this.httpClient.get<PaginatedResponse<number>>(`${this.cupRatingUrl}/group-furtherance-by-position`, { params });
```

Comment:
the path is `v2/cup/group-furtherance-by-position`, so we can't use `cupRatingUrl` here.